logs/040ded3d-9ed3-4e1a-a64f-c547b352c300.txt

```diff
@@ -0,0 +1 @@
+{'name': 'edx.notifications.forums.post.created', 'recipients': None, 'fields': None, 'current_step': 'notifications_pipeline_steps.NotificationsPipelineDelivery', 'expiration_time': datetime.datetime(2016, 9, 29, 13, 47, 32, 856454), 'id': UUID('040ded3d-9ed3-4e1a-a64f-c547b352c300'), 'history': [<NotificationsMessage.NotificationsPipelineHistory object at 0x10b919190>, <NotificationsMessage.NotificationsPipelineHistory object at 0x10b9191d0>, <NotificationsMessage.NotificationsPipelineHistory object at 0x10b919210>, <NotificationsMessage.NotificationsPipelineHistory object at 0x10b919250>, <NotificationsMessage.NotificationsPipelineHistory object at 0x10b919290>]}
```

Comment:
Create a useful `__repr__` method for the NPHistory class, so the serialized history is usefully readable.